src/status_im/contexts/settings/wallet/keypairs_and_accounts/style.cljs

```diff
@@ -1,7 +1,8 @@
 (ns status-im.contexts.settings.wallet.keypairs-and-accounts.style)
 
 (def title-container
-  {:padding-horizontal 20
+  {:flex               0
```

Comment:
Why `:flex 0`? (also I think the default value of flex is 0)

Reply:
We're using `:flex 0` here to override the `quo/standard-title` component default styling of `:flex 1`.